.gitignore

```diff
@@ -2,4 +2,4 @@
 *.pyc
 battlesnake
 .idea
-.vscode
+.vscode
```

Comment:
just ignore this one I changed my mind about something and couldn't figure out how to cleanly remove it from history